go/interpreter/lfvm/interpreter_test.go

```diff
@@ -721,17 +703,7 @@ func benchmarkFib(b *testing.B, arg int, with_super_instructions bool) {
 			b.Fatalf("execution failed: status is %v", ctxt.status)
 		}
 
-		size := ctxt.resultSize
-		if size.Uint64() != 32 {
-			b.Fatalf("unexpected length of end; wanted 32, got %d", size.Uint64())
-		}
-		res := make([]byte, size.Uint64())
-		offset := ctxt.resultOffset
-
-		data, err := ctxt.memory.getSlice(offset.Uint64(), size.Uint64(), &ctxt)
-		if err != nil {
-			b.Fatalf("unexpected error: %v", err)
-		}
+		res := make([]byte, len(ctxt.returnData))
```

Comment:
Why make a new buffer and copy the `ctxt.returnData` instead of just using the `ctxt.returnData`?